java/vector/src/test/java/org/apache/arrow/vector/TestDictionaryVector.java

```diff
@@ -870,16 +775,8 @@ public void testEncodeStructSubField() {
       DictionaryProvider.MapDictionaryProvider provider = new DictionaryProvider.MapDictionaryProvider();
 
 
-      dictVector1.setSafe(0, "aa".getBytes(StandardCharsets.UTF_8));
-      dictVector1.setSafe(1, "bb".getBytes(StandardCharsets.UTF_8));
-      dictVector1.setSafe(2, "cc".getBytes(StandardCharsets.UTF_8));
-      dictVector1.setSafe(3, "dd".getBytes(StandardCharsets.UTF_8));
-      dictVector1.setValueCount(4);
-
-      dictVector2.setSafe(0, "foo".getBytes(StandardCharsets.UTF_8));
-      dictVector2.setSafe(1, "baz".getBytes(StandardCharsets.UTF_8));
-      dictVector2.setSafe(2, "bar".getBytes(StandardCharsets.UTF_8));
-      dictVector2.setValueCount(3);
+      setVector(dictVector1,"aa".getBytes(), "bb".getBytes(), "cc".getBytes(), "dd".getBytes());
```

Comment:
we should either include the character set here of make use just make  the use of "aa" possible.

Reply:
Thanks, fixed.